scripts/db/populate_departments.sql

```diff
@@ -183,24 +183,26 @@ WITH listings (dept_name, subject_area, catalog_id) AS (VALUES
   ('Education', 'SCMATHE', NULL),
   ('Electrical Engineering and Computer Sciences', 'COMPSCI', NULL),
   ('Electrical Engineering and Computer Sciences', 'EECS', NULL),
-  ('Electrical Engineering and Computer Sciences', 'EL ENG', NULL),
+  ('Electrical Engineering and Computer Sciences', 'ELENG', NULL),
   ('Energy and Resources Group', 'ENE,RES', NULL),
   ('Engineering', 'AST', NULL),
   ('Engineering', 'DES INV', NULL),
   ('Engineering', 'ENGIN', NULL),
   ('Engineering', 'NSE', NULL),
   ('English', 'CRWRIT', NULL),
   ('English', 'ENGLISH', NULL),
-  ('Environmental Science, Policy and Management', 'ENV SCI', NULL),
+  ('Environmental Science, Policy and Management', 'ENVSCI', NULL),
   ('Environmental Science, Policy and Management', 'ESPM', NULL),
   ('Ethnic Studies', 'ASAMST', NULL),
   ('Ethnic Studies', 'CHICANO', NULL),
-  ('Ethnic Studies', 'ETH GRP', NULL),
-  ('Ethnic Studies', 'ETH STD', NULL),
+  ('Ethnic Studies', 'ETHGRP', NULL),
+  ('Ethnic Studies', 'ETHSTD', NULL),
   ('Ethnic Studies', 'NATAMST', NULL),
   ('Film and Media', 'FILM', NULL),
   ('French', 'FRENCH', NULL),
-  ('Freshman and Sophomore Seminars', 'FSSEM', NULL),
+  ('Freshman and Sophomore Seminars', '', '(39|24|84)[A-Z]*'),
+  ('Freshman and Sophomore Seminars', 'MCELLBI', '90[A-Z]'),
+  ('Freshman and Sophomore Seminars', 'NATAMST', '90[A-Z]'),
```

Comment:
Rather than keep around the not-actually-a-subject-area FSSEM notation from old OEC, we upgrade the loch fetch to handle regular expressions in catalog ids. The `(39|24|84)[A-Z]*` will match against courses in any subject area.